### PR TITLE
change format in messages and outputs

### PR DESCRIPTION
**Pull request by jzuniga-amd**

### CODE REVIEW

File: clients/benchmarks/client.cpp [hunk 1]
```diff
@@ -22,60 +22,81 @@ try
...
     options_description desc("rocsolver client command line options");
-    desc.add_options()("help,h", "produces this help message")
+    desc.add_options()("help,h", "produces this help message.")
 
         // test options
         ("function,f",
          value<std::string>(&function)->default_value("potf2"),
-         "The LAPACK function to test.")
+            "The LAPACK function to test.\n"
+            "                           Options are: getf2, getfr, gesvd_batched, etc.\n"
```
Comment (tfalders) [inline clients/benchmarks/client.cpp:33]: Spelling error on getrf

File: clients/rocblascommon/program_options.hpp [hunk 2]
```diff
@@ -377,10 +377,21 @@ class options_description
...
     {
+        os << "\nrocSOLVER benchmark client help.\n\n" << "Usage: ./rocsolver-bench <options>\n\n" 
+            << "In addition to some common general options, the following list of options corresponds to all the parameters\n"
+            << "that might be needed to test a given rocSOLVER function. The parameters are named as in the API user guide.\n"
+            << "The arrays are initialized internally by the program with random values.\n\n" 
+            << "Note: When a required parameter/option is not provided, it will take the default value as listed below.\n"
+            << "If no default value is defined, the program will try to calculate a suitable value depending on the context\n"
+            << "of the problem and the tested function; if this is not possible, the program will abort with error.\n\n"   
+            << "Example: ./rocsolver-bench -f getf2_batched -m 30 --lda 75 --batch_count 350\n"
+            << "This will test getf2_batched with a set of 350 random 30x128 matrices. strideP will be set to be equal 30.\n\n"
```
Comment (tfalders) [inline clients/rocblascommon/program_options.hpp:388]: Correction: equal to 30

**Review by tfalders — APPROVED** [review]
LGTM! Let me know once you want it merged in.

**Comment by jzuniga-amd** [comment]
Ready to merge if you agree...

**Pull request merged**